python/aitemplate/frontend/nn/attention.py

```diff
@@ -395,7 +419,8 @@ def forward(self, *args):
         else:
             x = self.proj(attn_output)
         x = self.proj_drop(x)
-        x = ops.reshape()(x, [batch, -1, self.dim])
+        if not isinstance(batch, IntVar):
+            x = ops.reshape()(x, [batch, -1, self.dim])
```

Comment:
This looks weird. Why this change?

Reply:
ROCM attention support [batch * seqlen, nheads] tensor as inputs and output [batch * seqlen, nheads] tensor. The branch keep original behavior.